src/lib.rs

```diff
@@ -1,3 +1,20 @@
+//! An implementation of the server side of the GDB Remote Serial
+//! Protocol -- the protocol used by GDB and LLDB to talk to remote
+//! targets.
+//!
+//! This library attempts to hide many of the protocol warts from
+//! servers.  It is also mildly opinionated, in that it implements
```

Comment:
Are we hiding them from servers (which this library is for?) or from implementers of servers?

Reply:
I changed it to read "from server implementations".